UI/src/view/LinearAssetForm.js

```diff
@@ -125,9 +125,18 @@
   }
 
   var renderLinktoWorkList = function renderLinktoWorkList(layerName) {
+    var textName;
+    switch(layerName) {
```

Comment:
this probably could be changed to map, because in the future we may need to switch for more layers